01-filedir.md

```diff
@@ -336,7 +336,10 @@ As you can see,
 it also displays another special directory that's just called `.`,
 which means "the current working directory".
 It may seem redundant to have a name for it,
-but we'll see some uses for it soon.
+but we'll see some uses for it soon.  
+Finally, we also see a file called `.bash_profile`. This file usually contains settings to customize the shell (terminal). There may also be similar files called `.bashrc` or `.bash_login`. For this lesson material it does not contain any settings.
```

Comment:
I agree with your point on 'workshop settings'.

I do think that it is helpful to 'new' and 'novice' users to explicitly state that without the '-a' option the dotfile will not be displayed. Although an experienced user may find it redundant, many new learners will need the cognitive reinforcement.